app/views/handlers/feedback.py

```diff
@@ -174,3 +199,45 @@ def is_limit_reached(session_data: SessionData) -> bool:
     @staticmethod
     def is_enabled(schema: QuestionnaireSchema) -> bool:
         return schema.get_submission().get("feedback")
+
+
+class FeedbackMetadata:
+    def __init__(self, feedback_count, form_type, language_code, region_code, tx_id):
+        self.feedback_count = feedback_count
+        self.form_type = form_type
+        self.language_code = language_code
+        self.region_code = region_code
+        self.tx_id = tx_id
```

Comment:
Have you looked at whether we need timestamp based on https://github.com/ONSdigital/eq-questionnaire-runner/pull/416#discussion_r524527678
As if Big Query just load JSON data from GCS then I don't think it will have access to the "blobs" attributes.

Reply:
Yep, I have now had it confirmed to add feedback submission date and done so